src/actions/webhook.js

```diff
@@ -53,7 +54,7 @@ var addWebhookService = function addWebhook(client, args, webhook) {
 
 	this.populateBody()
 
-	var hash = murmur.hash128(JSON.stringify(this.query)).hex()
+	var hash = murmur.x86.hash128(stringify(this.query));
```

Comment:
murmur package was the main reason why there was bug. It was returning the same hash for different object. Changing it to any other hashing library will solve this problem.There is x64 architecture support as well.